modules/ezfind/module.php

```diff
@@ -42,6 +42,14 @@
     'params' => array( 'ObjectID', 'SearchQuery', 'LanguageCode' )
                                     );
 
+$ViewList[ 'query' ] = array(
+    'functions' => array( 'elevate' ),
```

Comment:
Since this isn't related to elevation, this should get its own function 'query_admin' or similar.